bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/service.js

```diff
@@ -24,17 +24,35 @@ const clearWhiteboard = (whiteboardId) => {
   makeCall('clearWhiteboard', whiteboardId);
 };
 
+const reloadWhiteboard = (whiteboardId) => {
+  makeCall('reloadWhiteboard', whiteboardId);
+};
+
 const changeWhiteboardMode = (multiUser, whiteboardId) => {
-  makeCall('changeWhiteboardAccess', multiUser, whiteboardId);
+  const whiteboardMultiuserData = WhiteboardMultiUser.findOne({ meetingId: Auth.meetingID, whiteboardId });
+  const multiUser_p = whiteboardMultiuserData ? whiteboardMultiuserData.multiUser : 0;
+  
+  if ( (multiUser_p == 2 && (multiUser   == 1 || multiUser   == 0)) ||
+       (multiUser   == 2 && (multiUser_p == 1 || multiUser_p == 0)) ) {
+    async function syncChangeWhiteboardAccess () {
+      await makeCall('changeWhiteboardAccess', multiUser, whiteboardId);
+      // In rare cases, drawing still starts before the multiUser change reaches...
+      reloadWhiteboard(whiteboardId);
```

Comment:
This still has a problem, although very rarely (about once out of 20 multi-user mode changes).
The problem is generated when the annotation reloading starts before the makeCall('changeWhiteboardAccess') finishes. If I wait, say, 3 seconds to call reloadWhiteboard() after makeCall('changeWhiteboardAccess'), it always works...